bin/all_sky_search/pycbc_fit_sngls_split_binned

```diff
@@ -238,6 +238,9 @@ if args.gating_veto_windows:
     if not (gveto_before == 0 and gveto_after == 0):
         time = trigf[args.ifo + '/end_time'][:]
         gate_times = np.unique(trigf[args.ifo + '/gating/auto/time'][:])
+        if args.ifo is not 'V1':
```

Comment:
We should not assume that H and L have file gates and V never has them, eg V1 could produce file gates in future. Instead, check if the ifo/gating/file/ group exists and if so read the times.